pygments/lexers/procfile.py

```diff
@@ -0,0 +1,40 @@
+"""
+    pygments.lexers.procfile
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for Procfile file format.
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Name, Number, String, Text, Punctuation
+
+__all__ = ["ProcfileLexer"]
+
+
+class ProcfileLexer(RegexLexer):
+    """
+    Lexer for Procfile file format.
+
+    The format is used to run processus on Heroku or is used by Foreman or
```

Comment:
Is this a typo? "processus"?

Reply:
Sorry, it's a typo. It's the french word for `processes`. It's fixed.

Reply:
TIL :) Thanks!